src/tools/PrizeTierController/SavePrizeTiersModal/DrawIdForm.tsx

```diff
@@ -0,0 +1,74 @@
+import { ErrorMessage } from '@components/ErrorMessage'
+import { StyledInput } from '@components/Input'
+import { Label } from '@components/Label'
+import { useEffect } from 'react'
+import { FieldErrorsImpl, useForm, UseFormRegister } from 'react-hook-form'
+
+export const DrawIdForm = (props: {
+  onChange: (drawId: number) => void
+  defaultValues: { drawId: string }
+  minDrawId: number
+}) => {
+  const { onChange, defaultValues, minDrawId } = props
+  const {
+    register,
+    watch,
+    formState: { errors, isValid }
+  } = useForm<{ drawId: string }>({
+    mode: 'onChange',
+    defaultValues,
+    shouldUnregister: true
+  })
+
+  const drawId = watch('drawId')
+
+  useEffect(() => {
+    const numDrawId = Number(drawId)
+    if (Number.isInteger(numDrawId) && numDrawId > minDrawId) {
+      onChange(numDrawId)
+    }
+  }, [drawId])
+
+  return (
+    <form className='flex flex-col' autoComplete='off'>
+      <DrawIdInput minDrawId={minDrawId} errors={errors} register={register} />
+    </form>
+  )
+}
+
+const DrawIdInput = (props: {
+  minDrawId: number
+  errors: FieldErrorsImpl<{ drawId: string }>
+  register: UseFormRegister<{ drawId: string }>
+}) => {
+  const { minDrawId, errors, register } = props
+
+  return (
+    <div>
+      <Label className='uppercase' htmlFor='drawId'>
```

Comment:
Some more info on what this draw id is would be helpful. A small description sentence along the lines of  "new Prize Tier begins on draw:" would do the trick.

Reply:
Added in 3ec51e3